tensorflow/lite/micro/tools/requantize_flatbuffer_utils.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import numpy as np
+from absl import logging
+
+MIN_INT8, MAX_INT8 = -128, 128  # Wide
+
+MIN_INT16, MAX_INT16 = -32767, 32767  # Narrow range
```

Comment:
likewise, confusing that we have a different range here. probably worth a comment, and perhaps seeing if it can be narrowed in scope (i.e. not a global).

Reply:
MAX_INT16 seems to be unused?

Reply:
Same as the last comment. I will use [-32768, 32767] then